assets/built-in/transformers/containerimagespushscript/templates/pushimages.sh

```diff
@@ -15,17 +15,21 @@
 
 # Invoke as pushimages.sh <registry_url> <registry_namespace>
 {{ $containerRuntime := .ContainerRuntime }}
-if [ "$#" -ne 2 ]; then
-    REGISTRY_URL={{ .RegistryURL }}
-    REGISTRY_NAMESPACE={{ .RegistryNamespace }}
-else
+
+REGISTRY_URL={{ .RegistryURL }}
+REGISTRY_NAMESPACE={{ .RegistryNamespace }}
+if [ "$#" -eq 2 ]; then
     REGISTRY_URL=$1
     REGISTRY_NAMESPACE=$2
 fi
 
 # Uncomment the below line if you want to enable login before pushing
 # {{ $containerRuntime }} login ${REGISTRY_URL}
```

Comment:
Do we want to allow taking the runtime as a n input, say through environment variable pr something so that it can be overridden if required by user.This way we can avoid taking it as an input from user too.

Reply:
Yes sure, I have opened a PR for this https://github.com/konveyor/move2kube/pull/900. Please review it.